internal/peer/redis.go

```diff
@@ -179,11 +179,11 @@ func buildOptions(c config.Config) []redis.DialOption {
 	useTLS, _ := c.GetUseTLS()
 	if useTLS {
 		tlsConfig := &tls.Config{
-			MinVersion: tls.VersionTLS12,
+			MinVersion:         tls.VersionTLS12,
+			InsecureSkipVerify: true,
```

Comment:
```suggestion
```

Thanks for catching that skip verify was a noop! We came to the conclusion that it ought not to have been set in the first place, so let's just remove the noop line, and allow it to keep verifying as it is doing today.

Reply:
Ah, understood. We thought it was just something ya'll noticed while bumping the dependency. If you'd like, we can just merge the dependency change in this PR, and open a new issue for the skipVerify? On the issue, can you provide some context for your use case?

Reply:
@vreynolds Removed the code for InsecureSkipVerify from this PR. And will open an issue